### ISSUE TITLE
Mulitple design tabs?

### ISSUE THREAD

**Issue by cgatian** [pocka/storybook-addon-designs#7]
Perhaps you have a PDF and Figma attached to a story. It would be cool to allow multiple Design tabs for each type.

**Comment by pocka** [comment]
@cgatian 
Yes, it sounds great!

I believe this isn't so hard to implement.
Would API be something like this?

```js
.add('foo', () => <Foo/>, {
  design: [
    { type: 'figma', url: 'figma-file-1' },
    { type: 'pdf', url: 'my-pdf' },
    // Tab(Select box) name might be needed
    { type: 'figma', url: 'figma-file-2', name: 'My Another Figma File' }
  ]
})
// Also accepts current object style parameter for backwards compatibility
.add('bar', () => <Bar/> {
  design: {
    type: 'figma',
    url: 'figma-url'
  }
})
```

**Comment by cgatian** [comment]
Yes!

**Comment by pocka** [comment]
Implemented in v5.1.0 ([Preview](https://pocka.github.io/storybook-addon-designs/?path=/story/examples-advanced--embed-multiple-designs))
